backend/controller/console/console.go

```diff
@@ -188,6 +153,229 @@ func (c *ConsoleService) GetModules(ctx context.Context, req *connect.Request[pb
 	}), nil
 }
 
+func moduleFromDeployment(deployment dalmodel.Deployment, sch *schema.Schema) (*pbconsole.Module, error) {
+	module, err := moduleFromDecls(deployment.Schema.Decls, sch)
+	if err != nil {
+		return nil, err
+	}
+
+	module.Name = deployment.Module
+	module.DeploymentKey = deployment.Key.String()
+	module.Language = deployment.Language
+	module.Schema = deployment.Schema.String()
+
+	return module, nil
+}
+
+func moduleFromDecls(decls []schema.Decl, sch *schema.Schema) (*pbconsole.Module, error) {
```

Comment:
Done! I only added an integration test for now and didn't test for the contents of the various `<blahblah>FromDecl` methods since we're going to change so much of that anyway.